modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,269 @@
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+ name: sriov-network
+ namespace: openshift-sriov-network-operator
+spec:
+ networkNamespace: test-namespace
+ resourceName: sriovnic
+ spoofChk: "off"
+ trust: "on"
+----
++
+[NOTE]
+=====
+See the "Configuring SR-IOV additional network" section for a detailed explanation on each option in `SriovNetwork`.
+=====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a container's parent pod.
+
+. Create the `SriovNetwork` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-network-attachment.yaml
+----
+
+. Create a YAML file `tap-example.yaml` defining a network attachment definition, such as in the following example:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+ name: tap-one
+ namespace: test-namespace <1>
+spec:
+ config: '{
+   "cniVersion": "0.4.0",
+   "name": "tap",
+   "plugins": [
+     {
+        "type": "tap",
+        "multiQueue": true,
+        "selinuxcontext": "system_u:system_r:container_t:s0",
+        "owner": 1001,
+        "group": 1001
+     },
+     {
+       "type":"tuning",
+       "capabilities":{
+         "mac":true
+       }
+     }
+   ]
+ }'
+----
+<1> Specify the same `target_namespace` where the `SriovNetwork` object is created.
+
+. Apply the `yaml` by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f tap-example.yaml
+----
+
+. Create a performance profile by following the guidance in "Creating a performance profile".
+
+. Create the following `Pod` spec, and then save the YAML in the `dpdk-pod-rootless.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  namespace: test-namespace <1>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {"name": "sriov-network", "namespace": "test-namespace"},
+      {"name": "tap-one", "interface": "ext0", "namespace": "test-namespace"}]'
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: "worker-1"
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+  - name: testpmd
+    image: <DPDK_image> <2>
+    securityContext:
+      capabilities:
+        drop: ["ALL"] <3>
+        add: <4>
+          - IPC_LOCK
+          - SYS_RESOURCE
+          - NET_RAW #for mlx only <5>
+          - NET_ADMIN
+      fsGroup: 801 <6>
+      runAsUser: 1001 <7>
+      runAsGroup: 1001 <8>
+      privileged: false <9>
+      allowPrivilegeEscalation: true <10>
+      runAsNonRoot: true <11>
+    volumeMounts:
+    - mountPath: /dev/hugepages <12>
+      name: hugepage
+    resources:
+      limits:
+        openshift.io/mlxnics: "1" <13>
+        memory: "1Gi"
+        cpu: "4" <14>
+        hugepages-1Gi: "4Gi" <15>
+      requests:
+        openshift.io/mlxnics: "1"
```

Comment:
Please update to **openshift.io/sriovnic** according to policy that we added on previous step.